src/map/mob.c

```diff
@@ -3965,9 +4238,18 @@ static void mob_read_db_mvpdrops_sub(struct mob_db *entry, struct config_setting
 			i++;
 			continue;
 		}
-		if (mob->get_const(drop, &i32) && i32 >= 0) {
-			value = i32;
+		
+		struct optdrop_group *drop_option = NULL;
+		if (config_setting_is_number(drop)) {
+			// Setting is a number, item doesn't contain options
+			if (mob->get_const(drop, &i32) && i32 >= 0) {
```

Comment:
What happens if i32 is a negative value? perhaps throwing an error and put a default value.

Reply:
`value` by default is 0 (a couple lines above), if `i32` is negative, `value` will stay on 0, and on line 4253 (a couple lines below) there is a check for value <= 0 where it throws a warning that this drop chance is invalid. Keeping it like that will handle both if/else cases of `is_number` check in the same place.